bazel/es2k.bzl

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018-present Open Networking Foundation
+# Copyright 2022 Intel Corporation
```

Comment:
Fix date here

Reply:
The file was lifted verbatim from the Stratum repository. It's not new IP.